cmd/brimcap/load/command.go

```diff
@@ -23,10 +23,15 @@ import (
 
 var Load = &charm.Spec{
 	Name:  "load",
-	Usage: "load [options] [ pcapfile ]",
-	Short: "",
-	Long:  ``,
-	New:   New,
+	Usage: "load [options] pcap",
+	Short: "analyze a pcap and send logs into the Brim desktop client",
+	Long: `
+The load command is the same as the analyze command expect the output stream of
+generated logs is written to a specified space in the Brim desktop client.
```

Comment:
Strictly speaking it's writing to the `zqd` server that's running behind the Brim desktop client, right?

Reply:
Yes, I was catering to the unsophisticated user here.

Reply:
Ok. My guess is that the user who drops to the CLI and starts looking at `brimcap help` is likely to be the more sophisticated type of user, whereas I expect the less-sophisticated user is going to just keep doing drag & drop in the app and be largely unaware of what Brimcap is doing under the covers. But we can keep improving the wording over time.